mozilla_django_oidc_db/settings.py

```diff
@@ -1,7 +0,0 @@
-# The cache that should be used, e.g. 'default'. Refers to Django CACHES setting.
-# Set to None to disable caching.
-MOZILLA_DJANGO_OIDC_DB_CACHE = "oidc"
-
-MOZILLA_DJANGO_OIDC_DB_CACHE_TIMEOUT = 60 * 5
```

Comment:
Since this setting is removed, it can only be configured using `SOLO_CACHE_TIMEOUT`, right? Could you change the reference of `MOZILLA_DJANGO_OIDC_DB_CACHE_TIMEOUT` in the docs to `SOLO_CACHE_TIMEOUT`?

Reply:
yep!